README.md

```diff
@@ -0,0 +1,92 @@
+### BOOKSTORE
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+## About the Project
+
+> This project is built with React, it helps organise and save your favorite books.
+
+## Built With
+
+- React.js
+- Jest
+- Gitflow
+
+### Key Features
+
+- Page Setup
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link](https://saluumaa.github.io/book-store/)
```

Comment:
- [ ] I noticed that this live link leads to an error page. If your project has not yet been deployed, then please replace the`
[Live Demo Link](https://saluumaa.github.io/book-store/)` with `N/A` or `Coming Soon` instead of including a false link in your README :+1: 

![image](https://user-images.githubusercontent.com/102699958/230038737-096f5632-789c-4a59-840b-f9d6542198e2.png)